image.yaml

```diff
@@ -0,0 +1,33 @@
+name: "jboss-amq-7/amq70"
+description: "Red Hat JBoss AMQ 7.0 container image"
+version: "7.0.1"
+from: "jboss/openjdk18-rhel7:1.0"
+user: 185
+envs:
+    - name: "JBOSS_PRODUCT"
+      value: "AMQ"
+    - name: "JBOSS_AMQ_VERSION"
+      value: "7.0.1.GA"
+    - name: "PRODUCT_VERSION"
+      value: "7.0.1.GA"
+    - name: "AMQ_HOME"
+      value: "/opt/amq"
+cmd:
+    - "/home/jboss/broker/bin/artemis"
+    - "run"
+scripts:
+    - package: amq-install
+      exec: install.sh
+    - package: amq-chown
+      exec: install.sh
+sources:
+    - artifact: amq-broker-7.0.1.CR1-bin.zip
```

Comment:
In such case, `hint` should point to: https://access.redhat.com/jbossnetwork/restricted/softwareDetail.html?softwareId=52081&product=jboss.amq&version=7.0.1&downloadType=distributions